Dockerfile

```diff
@@ -245,9 +245,21 @@ USER buildbot
 RUN /bin/bash -c 'phpbrew init && source ~/.phpbrew/bashrc && phpbrew install 5.6 +default && \
     phpbrew app get composer'
 
+################################################################################
+#
+# Emacs Cask
+#
+################################################################################
 USER root
+RUN add-apt-repository ppa:kelleyk/emacs
```

Comment:
> v25.2 isn't all that different from v24.4

v25.2 has a **lot** of changes since 24.4. See [`NEWS.25`][1].

[1]: http://git.savannah.gnu.org/cgit/emacs.git/tree/etc/NEWS.25